### DISCUSSION TITLE
Welcome to inshackle-bot Discussions!

### DISCUSSION THREAD

**Discussion by cyberkallan** [cyberkallan/inshackle-bot#21] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by cyberkallan** [comment]
Error fixed

**Reply by Dimedulinux** [reply]
hello there your program is amazing but i have an issue with it doesnt increase the followers on the account i tried to install it again but it still doesnt give me followers please help me

**Comment by arjuncs911** [comment]
malyali pwoliyaaa

**Comment by investoraj1** [comment]
Is not working it doesnt increase the followers on the account
